tensorbay/opendataset/VOC2012ActionClassification/loader.py

```diff
@@ -12,7 +12,6 @@
 
 _SEGMENT_NAMES = (
     "train",
-    "trainval",
     "val",
```

Comment:
remove the `,` after `val`, `black` will format the `_SEGMENT_NAMES` into oneline